src/components/Loading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LoadingBar from 'react-redux-loading-bar';
+
+function Loading() {
+  return (
+    <div className="loading">
+      {/* @TODO: use react-redux-loading-bar to show loading bar */}
+      <LoadingBar />

```

Comment:
```suggestion
    <div className="loading">\
      <LoadingBar />

```
Jangan lupa untuk menghapus komentar yang sekiranya sudah tidak diperlukan lagi.